src/pages/AlgoliaPOC.vue

```diff
@@ -81,31 +124,75 @@ export default {
 		AisCurrentRefinements,
 		AisRefinementList,
 		AisHierarchicalMenu,
+		AisToggleRefinement,
 		AisBreadcrumb,
 		AisStats,
-		ActionButton,
+		KvCheckbox,
 		AlgoliaAdapter,
 		AisSortBy
 	},
+	inject: [
+		'apollo',
+	],
+	mixins: [
+		algoliaConfig,
+		algoliaInit
+	],
 	metaInfo: {
 		title: 'Algolia Search'
 	},
 	data() {
 		return {
+			// aka. Loan Channel config
```

Comment:
I'll probably remove these comments at some point and move them into a Confluence page, describing the options and how they work.